### PR TITLE
Update menu and pages to integrate new viz

### PR DESCRIPTION
**Pull request by critzo**
This PR makes modifications to the site to integrate the new M-Lab Viz site developed by Bocoup.

- Removes "Observatory" from the menu, but the Observatory page is still available.
- Keeps "Visualizations" as a menu item, but redirects it to https://viz.measurementlab.net
- Retains old visualizations sub-pages and places a text update at the top of them referencing the new viz:
-- /state
-- /transparency

The changes made in this PR can be [previewed on my fork](https://critzo.github.io/m-lab.github.io/).

**Note**: this PR should not be merged until the corresponding PR for the Viz site (altering menus and CSS there) is reviewed and merged.

---
This change is [Reviewable](https://reviewable.io/reviews/m-lab/m-lab.github.io/229)

### CODE REVIEW

**Comment by fernandalavalle** [comment]
LGTM

**Comment by georgiamoon** [comment]
@critzo is there a way to have the content of the current `/visualizations` page available on a page, e.g. `/visualizations/performance` since `/state` and `/transparency` had subpages, but `/performance` weirdly didn't?

**Comment by critzo** [comment]
@georgiamoon yes, I believe there is. I'll look at that change now. Any thoughts on the blog post and front page change (removed interconnection study, added the viz in the carosel)?

File: _posts/blog/2017-03-07-new-dataviz-site.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
...on Consumer Internet Performance]({{ site.baseurl }}/publications/M-Lab_Interconnection_Study_US.pdf), which made it possible for policy makers, researchers, journalists and others to monitor and analyze data about how interconnection points between internet service providers (ISPs) in the United States impact quality of experience. But M-Lab has much more data than just from the United States: it also has data from around the world and on many more ISPs, so we wanted to provide a tool that allowed anyone in the world to dig into our internet performance measurement test (NDT) data for their location. In 2016, we launched an initiative to revamp our visualizations and move to using open source technology to make them more flexible and auditable by the community. We worked with [Bocoup](https://bocoup.com/datavis){target="_blank"}, an open source technology and design consulting firm, to develop a data processing [pipeline](https://github.com/m-lab/mlab-vis-pipeline){target="_blank"}, [API](https://github.com/m-lab/mlab-vis-api){target="_blank"} and [visualizations](https://github.com/m-lab/mlab-vis-client){target="_blank"} all of which contribute to making the data more explorable.
```
Comment (georgiamoon) [inline _posts/blog/2017-03-07-new-dataviz-site.md]: The {target="_blank"} seems to be showing on the page?

Comment (georgiamoon) [inline _posts/blog/2017-03-07-new-dataviz-site.md]: for the one next to applications

Comment (critzo) [inline _posts/blog/2017-03-07-new-dataviz-site.md]: That should not be there. I'm missing a : at the start of those. Will add.

**Comment by georgiamoon** [comment]
LGTM!

**Pull request merged**